src/packages/admin-ui-components/src/title-bar/component.tsx

```diff
@@ -0,0 +1,67 @@
+import { Link, useSearchParams } from 'react-router-dom';
+import { Button } from '../button';
+import { Popover, PopoverItem } from '../popover';
+import OpenPlaygroundIcon from '../assets/16-open-external.svg';
+import { decodeSearchParams, routeFor, useSelectedEntity } from '../utils';
+import styles from './styles.module.css';
+
+interface Props {
+	title: string;
+	subtitle: string;
+	onExportToCSV: () => void;
+}
+
+// @todo allow to be extended
+const externalLinkItems: PopoverItem[] = [];
+
+export const TitleBar = ({ title, subtitle, onExportToCSV }: Props) => {
+	const [search] = useSearchParams();
+	const { filters, sort } = decodeSearchParams(search);
+	const { selectedEntity } = useSelectedEntity();
+
+	if (!selectedEntity) throw new Error('There should always be a selected entity at this point.');
```

Comment:
This is a pattern that's used throughout the codebase and it would break in many other places first, this is mostly so that Typescript will stop complaining.